sections/collage.liquid

```diff
@@ -213,6 +215,23 @@
       "default": "collage",
       "label": "t:sections.collage.settings.mobile_layout.label"
     },
+    {
+      "type": "select",
+      "id": "card_styles",
+      "options": [
+        {
+          "value": "",
+          "label": "t:sections.collage.settings.card_styles.options__1.label"
```

Comment:
Do we have instances where we use empty `value` for settings elsewhere?

Wouldn't it be better to use something like `none`, `individual` or something that explicitly tells us what that value is? No strong opinion on this, but it just looked weird to me.

Reply:
Yes we have it in the footer :) https://github.com/Shopify/dawn/blob/main/sections/footer.liquid#L458-L461

I find it better because we dont need to add a class when we dont have to